src/spot.sol

```diff
@@ -84,10 +84,8 @@ contract Spotter is DSNote {
     // --- Update value ---
     function poke(bytes32 ilk) external {
         (bytes32 val, bool zzz) = ilks[ilk].pip.peek();
```

Comment:
Could we change `zzz` to `has` if we're touching this function anyway? `zzz` usually denotes a time interval, not a boolean flag.

Reply:
Done!